docs/reference/mapping/types/geo-point.asciidoc

```diff
@@ -53,10 +53,16 @@ PUT my_index/_doc/4
   "location": [ -71.34, 41.12 ] <4>
 }
 
+PUT my_index/_doc/5
+{
+  "text": "Geo-point as a WKT POINT primitive",
+  "location" : "POINT (-77.03653 38.897676)" <5>
```

Comment:
Let's stick with the same coordinates that are used everywhere else in this example. They are not as cool as the White House, but since we flip lats and lons, consistency will make it easier to understand the effect of different formats on the positions of lats and lons.